login/login.c

```diff
@@ -314,24 +374,30 @@ int login_run(glome_login_config_t* config, const char** error_tag) {
   free(action);
   action = NULL;
 
-  printf("Obtain the one-time authorization code from:\n%s%s\n",
-         config->url_prefix, url);
-
+  size_t message_len =
+      strlen(prompt_format) + strlen(config->url_prefix) + strlen(url) - 4 + 1;
```

Comment:
I think it would be good to document somewhere why this works (the requirement for the format string is to contain precisely two `%s`?)

Reply:
Added a comment to the description of the `login_authenticate` function.